packages/@uppy/tus/src/index.js

```diff
@@ -229,9 +230,9 @@ module.exports = class Tus extends BasePlugin {
             done()
             return () => {}
           })
-          return p
+          return Promise.all([p, userProvidedPromise])
```

Comment:
Might be nice to add a comment here why we need two promises here?

Reply:
Do you have a suggestion what that comment would look like?